pkg/virt-controller/services/template.go

```diff
@@ -66,13 +65,13 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VirtualMachine) (*kubev1.P
 			"--qemu-timeout", "5m",
 			"--name", domain,
 			"--namespace", namespace,
-			"--socket-dir", t.socketBaseDir,
+			"--kubevirt-share-dir", t.virtShareDir,
```

Comment:
If I read this right, all virt-launchers now get a mount to the full shared directory for all VMs? Isn't that what you wanted to avoid for the sockets?